backend/routes/contractor.js

```diff
@@ -21,6 +21,9 @@ router.get('/profile/:contractor_id', getContractorProfile);
 router.use(authorizeContractor);
 
 // routes
+
+// Allows CO to view all listings in database (Filter feature included)
+router.post("/listings", viewMultipleListings );
```

Comment:
Should this be POST or get? I see that we send the filters we want to use, but we aren't actually changing any data on the server side. Would GET be more apt?

Reply:
Yes. This should be POST as GET method should not have a request body according to HTTP specifications. And we want a request body for specifying the filters.